backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/invoice/invoicePaymentAllocation.feature

```diff
@@ -1514,24 +1514,24 @@ Feature: invoice payment allocation
     And the invoice identified by customerCreditMemo1 is completed
 
     Then validate created invoices
-      | C_Invoice_ID        | C_BPartner_ID | C_BPartner_Location_ID | IsPaid | OpenAmt    | IsPartiallyPaid |
-      | vendorInvoice1      | bpartner1     | bpartner_location_1    | false  | 5.95       | false           |
-      | customerCreditMemo1 | bpartner2     | bpartner_location_2    | false  | grandtotal | false           |
+      | C_Invoice_ID        | C_BPartner_ID | C_BPartner_Location_ID | IsPaid | OpenAmt | IsPartiallyPaid |
+      | vendorInvoice1      | bpartner1     | bpartner_location_1    | false  | 5.95    | false           |
+      | customerCreditMemo1 | bpartner2     | bpartner_location_2    | false  | 59.5    | false           |
 
     And allocate sales credit memo to purchase invoice
       | C_Invoice_ID        |
       | vendorInvoice1      |
       | customerCreditMemo1 |
 
     And validate C_AllocationLines
-      | C_Invoice_ID        | Amount | OverUnderAmt | C_AllocationHdr_ID |
-      | vendorInvoice1      | -5.95  | 0            | alloc1             |
-      | customerCreditMemo1 | -5.95  | ???          | alloc1             |
+      | C_Invoice_ID        | Amount | C_AllocationHdr_ID |
+      | vendorInvoice1      | -5.95  | alloc1             |
+      | customerCreditMemo1 | -5.95  | alloc1             |
```

Comment:
What's the OverUnderAmt in manual testing?

Reply:
In manual testing, the overUnderAmt at this point is the sum of the grand total from the ARC and API , negated.

In this case, the manual test would have fulfilled this expectation: 

```
  And validate C_AllocationLines
      | C_Invoice_ID        | Amount | OverUnderAmt | C_AllocationHdr_ID |
      | vendorInvoice1      | -5.95  | 0            | alloc1             |
      | customerCreditMemo1 | -5.95  | -65.45       | alloc1             |
```

Reply:
how much is the OverUnderAmt of the C_AllocationLine generated by the cucumber test?

Reply:
With the current code, the numbers are: 
```
   And validate C_AllocationLines
      | C_Invoice_ID        | Amount | OverUnderAmt | C_AllocationHdr_ID |
      | vendorInvoice1      | -5.95  | -11.9        | alloc1             |
      | customerCreditMemo1 | -5.95  | -53.55       | alloc1           |
 ```